docker-compose.yml

```diff
@@ -13,19 +13,20 @@ services:
     links:
       - 'postgres:db'
       - redis
-    env_file: .env
+    env_file:
+      - .env
     environment:
       DJANGO_SETTINGS_MODULE: config.settings.local
       REDIS_URL: redis://redis:6379
     volumes:
       - .:/app
       - /app/node_modules
-      - ~/.gitconfig:/root/.gitconfig
     ports:
       - '0.0.0.0:8080:8080'
-        # Open this port to attach to remote debuggers in the Python process,
-        # if need-be:
-      - '0.0.0.0:33284:33284'  # Spells "DEBUG" in phone-number
+      - '0.0.0.0:1717:1717'  # For SFDX JWT Auth
```

Comment:
@wlonk So is this not actually necessary?

Reply:
At this moment I'm unsure, hold off until we can confirm with @davisagli .

Reply:
You need port 1717 open for completing the web server oauth flow, but shouldn't need it for the JWT flow.